src/ansys/fluent/core/docker/docker_compose.py

```diff
@@ -142,30 +141,36 @@ def _set_compose_cmds(self):
         return self._compose_cmds
 
     def check_image_exists(self) -> bool:
-        """Check if a Docker image exists locally."""
+        """Check if the image exists locally."""
         try:
-            output = subprocess.check_output(
-                ["docker", "images", "-q", self._image_name]
-            )
-            return output.decode("utf-8").strip() != ""
+            cmd = self._container_source + ["images", "-q", self._image_name]
+            # Podman users do not always configure rootless mode in /etc/subuids and /etc/subgids
+            if os.getenv("PYFLUENT_USE_PODMAN_COMPOSE") == "1":
```

Comment:
Please add a small method that hides this query for now. Additinal design improvements are possible.